### DISCUSSION TITLE
Automatic recipe import from images

### DISCUSSION THREAD

**Discussion by AntoineBriseboisRoy** [mealie-recipes/mealie#4260] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

As of now, we have the ability to convert images into recipe and it works great! 

However, we need to take a picture, then go in mealie, Create -> from image -> select image -> Create. It is a "long" process that could be so much easier in my opinion.

Or a better problem would be for bulk upload of images. I would be great to be able to drop all our recipes' picture into a folder and they all get imported.

### Please provide a concise description of the feature that would resolve your issue.

The solution to that is directory scan. A lot of people using mealie are doing so with their homeServer (assumption here) and a lot of those also happen to have a NAS (assumption also here). Wouldn't that be great, if for example, you can take a picture and put it in your recipe folder (or a temp_recipe folder), and mealie scan the directory continuously for new items and when there are, take the picture, make a call to the api for create recipe and then delete the item in the folder.

In my opinion, this is the last puzzle piece missing.

### Please consider and list out some caveats or tradeoffs made in your design decision

The only "caveat" that I see is that once mealie scan the folder, the be a little bit easier on the coding side, is to delete the file in the folder once processes. So you kinda lose the picture you took.

For me it's now an issue, but for some it could be.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by the1laz** [comment]
You could have a second folder for imported images, so the image files get moved over once they have been processed, if you need to keep the images

**Comment by inolen** [comment]
Would also be interested in this. Have a few cookbooks as PDFs that I enjoy, which I can split into individual images and would be great to then be able to bulk import.
